### PR TITLE
chore(macros/AvailableInWorkers): add zh-CN translations

### PR DESCRIPTION
**Pull request by skyclouds2001**
## Summary

relates to https://github.com/mdn/content/pull/36060 and https://github.com/mdn/yari/pull/11888

/cc @yin1999 @JasonLamv-t @jasonren0403 

### Solution

---

## Screenshots

observe the test result

### Before

None

### After

image

---

## How did you test this change?

manntually see browser preview

### CODE REVIEW

File: kumascript/macros/AvailableInWorkers.ejs [hunk 1]
```diff
@@ -57,6 +57,7 @@ const textNotService = mdn.localString({
 
 const textNotShared = mdn.localString({
   "en-US": `This feature is available in <a href="/${locale}/docs/Web/API/Web_Workers_API">Web Workers</a>, except for <a href="/${locale}/docs/Web/API/SharedWorkerGlobalScope">Shared Web Workers</a>.`,
+  "zh-CN": `此特性在 <a href="/${locale}/docs/Web/API/Web_Workers_API">Web Worker</a>（不包括 <a href="/${locale}/docs/Web/API/SharedWorkerGlobalScope">共享 Web Worker</a>）中可用。`,
```
Comment (yin1999) [inline kumascript/macros/AvailableInWorkers.ejs:60]:
```suggestion
  "zh-CN": `此特性在 <a href="/${locale}/docs/Web/API/Web_Workers_API">Web Worker</a>（不包括<a href="/${locale}/docs/Web/API/SharedWorkerGlobalScope">共享 Web Worker</a>）中可用。`,
```

**Review by yin1999 — COMMENTED** [review]
Let's remove the space

**Review by yin1999 — APPROVED** [review]
LGTM. Thank you @skyclouds2001

**Review by JasonLamv-t — APPROVED** [review]
👍🏼

**Review by caugner — APPROVED** [review]

**Pull request merged**